### PR TITLE
Feat/add custom url

### PR DESCRIPTION
**Pull request by kazerlelutin**

### CODE REVIEW

File: src/modules/se_scraper.js [hunk 1]
```diff
@@ -387,6 +386,18 @@ module.exports = class Scraper {
 
     }
 
+    get startUrl(){
+        const startUrls = {
```
Comment (HugoPoi) [inline src/modules/se_scraper.js]: I suggest to implement this as a static prop in each class like this :
```javascript
    defaultStartUrl = 'https://www.google.com';
    get startUrl(){
       return this.build_start_url(this.config.startUrl  || this.defaultStartUrl);
    }
```

File: src/modules/infospace.js [hunk 2]
```diff
@@ -41,11 +41,8 @@ class InfospaceScraper extends Scraper {
     }
 
     async load_start_page() {
-
-        let startUrl = this.build_start_url('http://search.infospace.com/search/web?') || 'http://infospace.com/index.html';
-
         try {
-            this.last_response = await this.page.goto(startUrl);
+            this.last_response = await this.page.goto(this.this.startUrl);
```
Comment (HugoPoi) [inline src/modules/infospace.js]: too much `this`

File: index.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ const se_scraper = require('./src/node_scraper.js');
 var Scraper = require('./src/modules/se_scraper');
 
 async function scrape(browser_config, scrape_config) {
+
```
Comment (HugoPoi) [inline index.js]: blank

**Review by HugoPoi — CHANGES_REQUESTED** [review]

**Comment by kazerlelutin** [comment]
@HugoPoi send with all fixes

**Review by HugoPoi — APPROVED** [review]

**Pull request merged**